docs/example_lambdas/sentiment.py

```diff
@@ -1,8 +1,3 @@
-"""
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-SPDX-License-Identifier: MIT-0
```

Comment:
Why stripped?

Reply:
It got stripped along with the similar.py statement, but I think this one should stay in. It is a direct path of derivation to get here